client/utils/charge/index.ts

```diff
@@ -37,8 +37,12 @@ export const isChargeCaptured = ( charge: Charge = <Charge>{} ): boolean =>
 export const isChargeDisputed = ( charge: Charge = <Charge>{} ): boolean =>
 	true === charge.disputed;
 
-export const isChargeRefunded = ( charge: Charge = <Charge>{} ): boolean =>
-	0 < charge.amount_refunded;
```

Comment:
Not sure if we need changes here at all, my guess is that this was needed due to changes in definitions.

Reply:
Yes, not needed, I'll revert them